test/run

```diff
@@ -68,10 +105,61 @@ def run_test(case, store):
     return True
 
 
+def get_local_boot_test_case(image_type: str) -> (str, Dict[Any, Any]):
+    with open(f"test/cases/{image_type}_local_boot.json", "r") as fd:
+        test_case_dict = json.load(fd)
+
+    pipeline_dict = test_case_dict["pipeline"]
+    return test_case_dict["compose"]["filename"], pipeline_dict
+
+
+def run_ssh_test(private_key):
+    cmd = ["ssh",
+           "-p", "1022",
+           "-i", private_key,
+           "-o", "StrictHostKeyChecking=no",
+           "redhat@localhost",
+           "systemctl is-system-running"]
+    for _ in range(40):
+        try:
+            sp = subprocess.run(cmd, timeout=120, check=True, stdout=subprocess.PIPE)
+            output = sp.stdout.decode('utf-8').strip()
+            print(output)
+            if output == "running":
+                print("ssh test success")
+                return 0
+        except subprocess.TimeoutExpired:
+            print("ssh timeout expired")
+        except subprocess.CalledProcessError as e:
+            print(f"ssh error: {e}")
+            time.sleep(20)
+
+    print("ssh test failure")
+    return 1
+
+
 def main(args):
+    # New part, this should be better organized
+    try:
+        flag: str = args[0]
+        if flag.startswith("--local-boot-test"):
```

Comment:
Maybe we should introduce some cmd line argument parsing, but we can do it as part of following PR.